packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -10,4 +10,37 @@ export class HdPaths {
     const iovPurpose = 4804438;
     return [Slip10RawIndex.hardened(iovPurpose), Slip10RawIndex.hardened(index)];
   }
+
+  /**
+   * Only use this for the real BIP-0044 with
```

Comment:
Like this.

Can we add a slip10hardened() function or something similar to generate ed25519 addresses?
I agree with a different name to differentiate them, but a similar helper for bns/lisk would be great.

Reply:
slip10 does not specify any path sheme. What we want is

```
trezorStyleBip44LikeAccountBasedEd25519(coinType: number, a: number)
```

which covers Stellar, Lisk, NEM, Monero as documented here https://github.com/trezor/trezor-core/tree/master/docs/coins as well as RISE and maybe others.

Anyway, we need to be much more specific in what kind of paths we need. For BNS no final path scheme other than simple address was specified.